pdm.lock

```diff
@@ -3,8 +3,9 @@
 
 [metadata]
 groups = ["default", "docs", "email", "linting", "memray", "mypy", "testing", "testing-extra"]
-strategy = ["cross_platform"]
-lock_version = "4.4"
+cross_platform = true
+static_urls = false
+lock_version = "4.3"
```

Comment:
Why was lock_version 'downgraded'?

Reply:
Good question - it shouldn't have been. I'll fix this in my next PR 👍. Thanks for pointing this out.

Reply:
@alexmojaki, see https://github.com/pydantic/pydantic/pull/8282/commits/833b9083b8099a52b68696fd82bcf548973a5f54